installation/generate_helm_values.py

```diff
@@ -302,11 +302,19 @@ def iter_helm_values() -> collections.abc.Generator[tuple[str, dict], None, None
         delivery_db_backup_enabled = 'deliveryDbBackup' in extensions
         freshclam_enabled = 'clamav' in extensions
 
+        delivery_service_cfg = cfg_set.delivery_service()
+
+        # inject rescoring cfg from delivery-service cfg to avoid duplication
+        rescoring_rule_sets = delivery_service_cfg.features_cfg()['rescoring']
```

Comment:
nit: I think the name is quite misleading because (at least for my understanding) this object is the umbrella object which contains the rescoring rule sets (next to some other meta information, like for example the default rescoring rule set). What about just `rescoring_cfg` or something similar?